### ISSUE TITLE
已经编译好版本的商品sku错了，大佬可以修改下重新发布吗？

### ISSUE THREAD

**Issue by duanyiGit** [zqjzqj/mtSecKill#10]

**Comment by zqjzqj** [comment]
sku没有错误 榨菜加入购物车操作只是用来获取一些必要的参数 并不会实际购买 如果想抢购其他商品 文档有运行参数 可以动态修改sku

**Comment by duanyiGit** [comment]
看到程序一直在加榨菜，如果要修改参数是在cmd命令端运行 mtSecKill.win.exe mtSecKill -sku=100012043978 -num=2 -works=6 -time=09:59:59 就可以了吗？不了解go语音，希望大佬可以解答下

**Comment by zqjzqj** [comment]
是的 一直加载榨菜应该是参数没有获取成功 会一直重新获取  可以重启一下试试或者你自己添加一点商品在你的购物车里并选中 期间最好不要手动去操作浏览器

**Comment by duanyiGit** [comment]
直接运行提示Enable colors error: The parameter is incorrect.应该就是参数获取不正确吧

**Comment by zqjzqj** [comment]
看起来像是你的操作系统或者cmd不太支持 换台电脑试试

**Comment by duanyiGit** [comment]
用powershell就可以了，感谢大佬
